spec/features/visitors/sign_up_spec.rb

```diff
@@ -255,19 +242,6 @@
     expect(current_path).to eq user_confirmation_path
   end
 
-  context 'confirmation instructions sent to existing user', email: true do
-    xit 'does not send an email to the existing user' do
```

Comment:
Before we remove it I'd like to see a resolution on the reason it was marked pending in the first place. We typically only mark tests as pending if they represent functionality that requires a business decision and we do not want to lose track of from an engineering perspective. The other example is deleting an account -- we have several pending tests for that feature because it was implemented, then yanked for a "wait and see" business decision.